src/Controllers/Setup/checks.php

```diff
@@ -183,79 +183,92 @@ function convertPHPSizeToBytes($sSize)
 ?>
 <!DOCTYPE html>
 <html>
-    <head>
-        <title>Welcome to MyRadio</title>
-        <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
-        <link rel="Stylesheet" type="text/css" href="css/normalise.css">
-        <link rel="Stylesheet" type="text/css" href="css/vendor/themes/ury-purple/jquery-ui.css">
-        <link rel="stylesheet" type="text/css" href="css/main.css">
-        <link rel="Stylesheet" type="text/css" href="css/style.css">
-        <script type="text/javascript" src="js/vendor/jquery-2.1.0.min.js"></script>
-    </head>
-    <body>
-      <div id="grid" class="transBG clearfix">
-          <header id="content-header"><h2>Hello there</h2></header>
-          <div id="content-body" class="clearfix">
-              <p>It looks like you're trying to install MyRadio! Would you like some help with that? No? Well too bad, I'm not a paperclip you can hide.</p>
-              <p>I'm just running some background checks to see if you're ready to go.</p>
-              <?php
-              if ($ready) {
-                  ?>
-                  <p class="ui-state-highlight">Good news! It looks like you're ready to go. <a href="?c=dbserver">Click here to continue</a>.</p>
-              <?php
-              } else {
-                  ?>
-                  <p class="ui-state-error">Uh oh! It looks like there's some things you'll have to get sorted out before you can continue. Follow the advice below, then <a href=''>refresh this page</a> to try again.</p>
-              <?php
-                  echo '<h3>The following tests failed and must be fixed before you can proceed:</h3><ul>';
-                  foreach ($problems as $problem) {
-                      echo '<li>'.$problem.'</li>';
-                  }
-                  echo '</ul>';
-              }
+  <head>
+    <title>Welcome to MyRadio</title>
+    <meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
+    <link rel="Stylesheet" type="text/css" href="css/vendor/bootstrap.min.css">
+    <link rel="Stylesheet" type="text/css" href="css/vendor/bootstrap-theme.min.css">
+    <link rel="Stylesheet" type="text/css" href="css/style.css">
 
-              if (empty($warnings)) {
-                  if ($ready) {
-                      echo '<p><span class="ui-icon ui-icon-circle-check fleft"></span>Amazing! Your server is absolutely <em>perfect</em> for running MyRadio.</p>';
-                  }
-              } else {
-                  echo '<h3>The following tests failed, but they aren\'t required for MyRadio to run:</h3><ul>';
-                  foreach ($warnings as $warning) {
-                      echo '<li>'.$warning.'</li>';
-                  }
-                  echo '</ul>';
-              }
+    <script type="text/javascript" src="js/vendor/jquery-2.1.0.min.js"></script>
+    <script type="text/javascript" src="js/vendor/bootstrap.min.js"></script>
```

Comment:
This is in setup so isn't the end of the world, and is how it did it before (although I do agree that should be moved) - not sure whether it's worth doing in this or in the later general JS refactor I have planned.